cmd/govim/gopls_client.go

```diff
@@ -165,6 +166,9 @@ func (g *govimplugin) Configuration(ctxt context.Context, params *protocol.Param
 	if g.vimstate.config.TempModfile != nil {
 		goplsConfig[goplsTempModfile] = *conf.TempModfile
 	}
+	if g.vimstate.config.Gofumpt != nil {
+		goplsConfig[goplsGofumpt] = *conf.Gofumpt
```

Comment:
I have to say I'm somewhat surprised they didn't opt for an "enum" here instead of a bool.

Reply:
Yeah. Could be due to the issue in gofumpt repo where it looks like the goal is to have gofumpt enabled by default, and provide diagnostics with suggested edits.